android/battleBuddyAndroid/src/screens/stacks/Learn/BallisticsScreen.js

```diff
@@ -1,52 +1,94 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import styled from 'styled-components/native';
-import Data from '../../../../test-data/ballistics';
 import ScrollableContainer from '../../../components/common/ScrollableContainer';
+import Image from '../../../components/learn/Image';
+import WebView from '../../../components/learn/WebView';
 
 const Text = styled.Text`
   color: ${({theme}) => theme.colors.white};
   text-align: left;
+  padding: 10px 20px;
 `;
 
 const MainTitle = styled(Text)`
-  font-size: 35px;
+  font-size: 35;
 `;
 
-const SubTitle = styled.Text`
+const SubTitle = styled(Text)`
   color: ${({theme}) => theme.colors.orange};
+  font-size: 20;
 `;
 
 const Bold = styled(Text)`
   font-weight: bold;
+  font-size: 20;
 `;
 
-const BallisticsScreen = () => {
+const Body = styled(Text)`
+  line-height: 20;
+`;
+
+const ScrollView = styled(ScrollableContainer)`
+  padding: 0 0;
+`;
+
+const BallisticsScreen = ({navigation}) => {
+  const {data} = navigation.state.params;
+
   return (
-    <ScrollableContainer>
-      {Data.map((post, index) => {
-        if (!post.title) {
+    <ScrollView>
+      {data.map((post, index) => {
+        if (post.maintitle) {
           return (
             <React.Fragment key={index}>
```

Comment:
Can you extract these Fragments (including below) into their own component? Would make the component a lot cleaner and contained. Something like PostImage PostVideo etc.

Reply:
Will do!